src/current/v23.2/install-cockroachdb-linux.md

```diff
@@ -167,8 +167,9 @@ true
 
 CockroachDB runtimes built for the ARM architecture have the following limitations:
 
-- CockroachDB on ARM is in [Limited Access](https://www.cockroachlabs.com/docs/stable/cockroachdb-feature-availability#feature-availability-phases) in CockroachDB v22.2.13, and is **experimental** in all other versions. Experimental binaries and Docker images are not qualified for production use and not eligible for support or uptime SLA commitments.
 - Clusters with a mix of Intel and ARM nodes are untested. Cockroach Labs recommends that all cluster nodes have identical CockroachDB versions, hardware, and software.
+- Because of the recommendation to avoid a mix of Intel and ARM nodes, Cockroach Labs recommends against migrating an Intel cluster to ARM in place. Instead, consider using CockroachDB's migration tooling, such as [Changefeed sinks]({% link v23.2/changefeed-sinks.md %}), to move the workload to a new ARM cluster.
```

Comment:
This is probably a question for @mwang1026, but methinks changefeed as a migration tool is a bit too narrow to be given as general guidance.

Reply:
Done, PTAL